p2p_wallet/Scenes/Main/Buy/BuyCoordinator.swift

```diff
@@ -143,12 +144,41 @@ final class BuyCoordinator: Coordinator<Void> {
             }
         }).store(in: &subscriptions)
 
-        viewModel.coordinatorIO.license.sink(receiveValue: { [weak self] url in
-            let vc = SFSafariViewController(url: url)
-            vc.modalPresentationStyle = .automatic
-            viewController.present(vc, animated: true)
-        }).store(in: &subscriptions)
-
+        viewModel.coordinatorIO.license
+            .sink(receiveValue: { url in
+                let vc = SFSafariViewController(url: url)
+                vc.modalPresentationStyle = .automatic
+                viewController.present(vc, animated: true)
+            })
+            .store(in: &subscriptions)
```

Comment:
please add new line after .store, so there will be more readable

Reply:
I think it's a not useful and doesn't improve readability of code. Minor comment